dask/highlevelgraph.py

```diff
@@ -87,6 +87,30 @@ def get_dependencies(self, all_hlg_keys: Iterable) -> Mapping[Hashable, Set]:
         """
         return {k: keys_in_tasks(all_hlg_keys, [v]) for k, v in self.items()}
 
+    def map_tasks(self, func: Callable[[Iterable], Iterable]) -> "Layer":
```

Comment:
This is just because `Layer` isn't declared at this point. Python allows using strings like `"Layer"` in order to _delay_ the evaluation to after `class Layer` has been declared. Semantically, it is the same.